specification/cognitiveservices/data-plane/AzureOpenAI/inference/stable/2023-05-15/generated.json

```diff
@@ -345,7 +345,7 @@
           "type": "array",
           "description": "The collection of context messages associated with this chat completions request.\nTypical usage begins with a chat message for the System role that provides instructions for\nthe behavior of the assistant, followed by alternating messages between the User and\nAssistant roles.",
           "items": {
-            "$ref": "#/definitions/ChatMessage"
+            "$ref": "#/definitions/ChatRequestMessage"
```

Comment:
The rename of the model appears to be retroactive. But only for `2023-05-01-stable`. Maybe this just needs reverting.